src/test/java/de/mediathekview/mserver/crawler/orf/tasks/OrfLetterPageTaskTest.java

```diff
@@ -0,0 +1,76 @@
+package de.mediathekview.mserver.crawler.orf.tasks;
+
+import de.mediathekview.mserver.crawler.orf.OrfTopicUrlDTO;
+import de.mediathekview.mserver.crawler.orf.OrfConstants;
+import de.mediathekview.mserver.testhelper.JsoupMock;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import org.hamcrest.Matchers;
+import org.jsoup.Jsoup;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Jsoup.class})
+public class OrfLetterPageTaskTest {
+
+  private final OrfTopicUrlDTO[] expectedUrls= new OrfTopicUrlDTO[] {
+    new OrfTopicUrlDTO("ABC Bär", "http://tvthek.orf.at/profile/ABC-Baer/4611813/ABC-Baer/13962996"),
+    new OrfTopicUrlDTO("ABC Bär", "http://tvthek.orf.at/profile/ABC-Baer/4611813/ABC-Baer/13962935"),
+    new OrfTopicUrlDTO("Ungarisches Magazin: Adj'Isten magyarok", "http://tvthek.orf.at/profile/AdjIsten-magyarok/13886441/Ungarisches-Magazin-AdjIsten-magyarok/13961067"),
+    new OrfTopicUrlDTO("Aktuell in Österreich", "http://tvthek.orf.at/profile/Aktuell-in-Oesterreich/13887571/Aktuell-in-Oesterreich/13962830"),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Aktuell in Österreich" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)